pkgs/development/libraries/agda/standard-library/default.nix

```diff
@@ -1,28 +1,26 @@
-{ stdenv, agda, fetchFromGitHub, ghcWithPackages }:
+{ stdenv, mkDerivation, fetchFromGitHub, ghcWithPackages }:
 
-agda.mkDerivation (self: rec {
-  version = "1.1";
-  name = "agda-stdlib-${version}";
+mkDerivation rec {
+  pname = "standard-library";
```

Comment:
I would have expected this to be called `stdlib`.

Reply:
The library seems to be referred to with different names everywhere, this is the version in the .agda-lib file

Reply:
Ok good point.